python/heterocl/devices.py

```diff
@@ -96,6 +97,19 @@ def __init__(self, cap=32, path="/dev/sda"):
         super(SSD, self).__init__("SSD", cap)
         self.path = path
 
+# Private memory to FPGA device
+class BRAM(Memory):
+    def __init__(self):
+        super(BRAM, self).__init__("BRAM", port=2)
+
+class LUTRAM(Memory):
+    def __init__(self):
+        super(LUTRAM, self).__init__("LUTRAM", port=2)
+
+class URAM(Memory):
+    def __init__(self):
+        super(URAM, self).__init__("URAM", port=2)
+
```

Comment:
If these are specific to FPGAs, should we introduce another hierarchy? Because we may support not only FPGAs.

Reply:
Maybe it's fine for now. But we will need to add one more level in the future.